### DISCUSSION TITLE
How are full-height arrows rendered in the status line?

### DISCUSSION THREAD

**Discussion by Yohnny777** [nvim-lualine/lualine.nvim#924] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/67546750/210102001-068033ed-d753-44d2-88eb-3dda67a91be4.png)

I assume it is some ASCII character, but I cannot seem to find which one it is. I've found arrows but they're not full-height:

![image](https://user-images.githubusercontent.com/67546750/210102139-bc291532-7358-4344-9c32-695aae85b848.png)

Any help appreciated

**Comment by yutkat** [comment, marked as answer]
Everyone should be using this.
https://github.com/nvim-lualine/lualine.nvim/blob/32a7382a75a52e8ad05f4cec7eeb8bbfbe80d461/README.md#L123


If it doesn't work, I think the font is wrong and you might try to change it.
